modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/model/SeqexecCircuit.scala

```diff
@@ -49,14 +50,28 @@ class SearchHandler[M](modelRW: ModelRW[M, Pot[SeqexecCircuit.SearchResults]]) e
 /**
   * Handles actions related to the search area, used to open/close the area
   */
-class SearchAreaHandler[M](modelRW: ModelRW[M, SearchAreaState]) extends ActionHandler(modelRW) {
+class SearchAreaHandler[M](modelRW: ModelRW[M, SectionVisibilityState]) extends ActionHandler(modelRW) {
   implicit val runner = new RunAfterJS
 
   override def handle = {
-    case OpenSearchArea =>
-      updated(SearchAreaOpen)
+    case OpenSearchArea  =>
+      updated(SectionOpen)
     case CloseSearchArea =>
-      updated(SearchAreaClosed)
+      updated(SectionClosed)
+  }
+}
+
+/**
+  * Handles actions related to the development console
+  */
+class DevConsoleHandler[M](modelRW: ModelRW[M, SectionVisibilityState]) extends ActionHandler(modelRW) {
+  implicit val runner = new RunAfterJS
+
+  override def handle = {
```

Comment:
Ok, I'll add types to them. In this particular case handle is `PartialFunction[AnyRef, ActionResult[M]]` where `M` is essentially our root model